Sources/Commands/SwiftPackageTool.swift

```diff
@@ -246,6 +254,24 @@ public class SwiftPackageTool: SwiftTool<PackageMode, PackageToolOptions> {
             // Put the dependency in edit mode.
             try workspace.edit(dependency: dependency, at: revision, packageName: manifest.name, checkoutBranch: options.editCheckoutBranch)
 
+        case .unedit:
+            guard options.enableNewResolver else {
+                fatalError("This mode requires --enable-new-resolver")
+            }
+            guard let packageName = options.packageName else {
+                throw PackageToolOperationError.insufficientOptions(usage: uneditUsage)
+            }
+            let workspace = try getActiveWorkspace()
+            let manifests = try workspace.loadDependencyManifests()
+            // Look for the package's manifest.
+            guard let manifest = manifests.lookup(packageName) else {
+                throw PackageToolOperationError.packageNotFound
+            }
+            guard let editedDependency = workspace.dependencyMap[RepositorySpecifier(url: manifest.url)] else {
+                fatalError("Unexpected failure, dependency for \(manifest.url) not found in workspace.")
+            }
```

Comment:
Minor cleanup, but since `loadDependencyManifests` only ever should return manifest which have `ManagedDependency` instances with them, I wonder if we should keep them together in that structure so clients don't need to do the double look + fatalError.